### PR TITLE
New Contrail API - Floating IP

### PR DESCRIPTION
**Pull request by jacekchalupka**

### CODE REVIEW

File: Test/Utils/ContrailAPI_New/Models/FloatingIpPool.ps1 [hunk 1]
```diff
@@ -0,0 +1,25 @@
+class FloatingIpPool : BaseResourceModel {
+    [String] $Name
+    [String] $NetworkName
```
Comment (mzaremba04) [inline Test/Utils/ContrailAPI_New/Models/FloatingIpPool.ps1]: Why isn't it NetworkFqdn? In FloatingIP there is PoolFqName, why is it different?

File: Test/Utils/ContrailAPI_New/Models/FloatingIp.ps1 [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+        $this.Name = $Name
+        $this.PoolFqName = $PoolFqName
+        $this.Address = $Address
+    }
+
+    [String[]] GetFqName() {
+        return ($this.PoolFqName + @($this.Name))
+    }
+
+    [Hashtable] GetRequest() {
```
Comment (mzaremba04) [inline Test/Utils/ContrailAPI_New/Models/FloatingIp.ps1]: What is the purpose of GetRequest command? Is it used for all of CRUD actions? I think that floating ip address is read-only property, so sending it for update will cause an error.

Comment (jacekchalupka) [reply Test/Utils/ContrailAPI_New/Models/FloatingIp.ps1]: I checked, it does not. You can update the floating_ip_address with in existing floating-ip with request like this.

File: Test/Utils/ContrailAPI_New/Models/FloatingIpPool.ps1 (hunk 1 above)
Comment (jacekchalupka) [reply Test/Utils/ContrailAPI_New/Models/FloatingIpPool.ps1]: Good point. Fixed. I need to work on my consistency in things like this. Thanks! :)

File: Test/Utils/ContrailAPI_New/Models/FloatingIp.ps1 (hunk 2 above)
Comment (mzaremba04) [reply Test/Utils/ContrailAPI_New/Models/FloatingIp.ps1]: Thanks for checking. It's weird, in schema this field is marked as CR so it misled me. 
Does API server allow changing the address? What if somebody tries to change the IP and then update the floating ip?

Comment (jacekchalupka) [reply Test/Utils/ContrailAPI_New/Models/FloatingIp.ps1]: Yeah, according to docs it is not field that can be passed when using PUT method (updating object). But I just did it, and even in WebUI IP changed and everything just worked fine :)

**Pull request merged**